pkg/scheduler/core/extender_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/algorithm"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
 	schedulercache "k8s.io/kubernetes/pkg/scheduler/cache"
+	internalqueue "k8s.io/kubernetes/pkg/scheduler/internal/queue"
```

Comment:
nit: should we just alias it as queue ?

Reply:
Not quite sure. I use `internalqueue` is because it's more self-descriptive than `queue`.

Reply:
I think `queue` and `internalqueue` are both fine. Today, `queue` is unambiguous in genericScheduler, but it's a generic word that could refer to something else.

The [golang style guide](https://github.com/golang/go/wiki/CodeReviewComments#imports) says,
> Avoid renaming imports except to avoid a name collision; good package names should not require renaming. In the event of collision, prefer to rename the most local or project-specific import.